routes/ci.js

```diff
@@ -322,7 +322,7 @@ var execDocker = function(buildId, buildItemId, accessToken, withSelenium) {
 
       // Predefine container options.
       var containerOptions = {
-        'Image': 'shoov/php-ci',
+        'Image': 'shoov/php-ci:' + conf_get('php_ci_docker_tag'),
```

Comment:
Why underscore instead of point? It is strange because in the master branch I see a point. 
`'Image': 'shoov/php-ci:' + conf.get('php_ci_docker_tag'),`